src/config/QuoteList.ts

```diff
@@ -97,4 +97,20 @@ export const QuoteList: Quote[] = [
     text: "Success is not final, failure is not fatal: it is the courage to continue that counts.",
     author: "Winston Churchill",
   },
+  {
+    text: "If you're walking down the right path and you're willing to keep walking, eventually you'll make progress.",
+    author: "Barack Obama",
+  },
+  {
+    text: "Where you are right now doesn't have to determine where you'll end up.",
+    author: "Barack Obama",
+  },
+  {
+    text: "You only live once,but if you do it right, once is enough",
```

Comment:
```suggestion
    text: "You only live once, but if you do it right, once is enough",
```
Just a quick spacing issue.